src/Native/CpuMathNative/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ set(SOURCES
 )
 
 if(NOT WIN32)
-    set_property(SOURCE Sse.cpp APPEND_STRING PROPERTY COMPILE_FLAGS " -msse3")
+    set_property(SOURCE Sse.cpp APPEND_STRING PROPERTY COMPILE_FLAGS " -msse3 -msse4.1 -msse4.2")
```

Comment:
Are we sure these build changes are necessary? This worked before on Ubuntu - what changed?